src/include/tcop/utility.h

```diff
@@ -16,6 +16,7 @@
 
 #include "tcop/cmdtag.h"
 #include "tcop/tcopprot.h"
+#include "parser/parse_node.h"
```

Comment:
why this header is required? specifically for which struct

Reply:
It is required for ParseState.

Reply:
please use forward declaration